.ruby-version

```diff
@@ -1 +1 @@
-2.3
+2.5.0
```

Comment:
Currently Login.gov servers run Ruby 2.3.x, and we don't have an easy mechanism to run different applications on different ruby versions.

Reply:
Ah, dang it, I'll drop it down.